Automattic/vip-go-mu-plugins
# Check plugin updates after every 6 hours

## Description


The Codebase Manager plugin keeps making POST requests to the WordPress' `update-check` (`ttps://api.wordpress.org/plugins/update-check/1.1/`) endpoint on every request.

In the code it was apparent that the developer intended to make such requests after every 6 hours, but instead of `<` they used `>` thereby ending up making those POST requests on every request instead.

## Changelog Description


This PR will ensure that sites using vip-go-mu-plugins will make requests to check for plugin updates after every 6 hours rather than on every request that they are doing currently.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

Before this PR:
1. Enable Query Monitor for your user
2. Open any `wp-admin` page (Dashboard page)
3. Check the `HTTP API Calls` section in the Query Monitor dashboard.
4. You will always see a POST request to `https://api.wordpress.org/plugins/update-check/1.1/` 
<img width="1470" alt="image" src="https://user-images.githubusercontent.com/1535505/209506609-b1fede91-3868-410d-ac1e-d31fabee9a00.png">


After this PR:
1. Repeat the same steps as mentioned above, but you will only see the request after every 6 hours.
